.rspec

```diff
@@ -1,3 +1,3 @@
 --color
 --require rails_helper
---format=documentation
```

Comment:
Was deleting this new line and intended consequence of this PR? (not a big deal at all, and I dont have strong feelings on it, but adding and [editorconfig](https://editorconfig.org/) might help reduce these kinds of unintended diffs)

Reply:
shit -- no it wasn't.  Thanks for catching that